src/components/notification-snackbar/NotificationSnackbar.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { message } from '..';
+import { IconCheckmark, IconError, IconStarOutlined, IconWarning } from '../..';
+import { SFlex } from '../../base-components/flex/styles/SFlex';
+import { SIcon } from './styles/SIcon';
+import { SIconError } from './styles/SIconError';
+import { SIconSuccess } from './styles/SIconSuccess';
+import { SNotificationSnackbar } from './styles/SNotificationSnackbar';
+import { SNotificationSnackbarAction } from './styles/SNotificationSnackbarAction';
+import { SNotificationSnackbarLabel } from './styles/SNotificationSnackbarLabel';
+
+export interface INotificationSnackbarProps {
+  exit: boolean;
+  icon?: React.ReactNode;
+  action?: React.ReactNode;
+  content: String;
+  success?: boolean;
+  error?: boolean;
+}
+
+export const NotificationSnackbar: React.FC<INotificationSnackbarProps> = ({ ...props }) => {
```

Comment:
@steiningermona As we discussed earlier today: destruct the props to be the ones in the interface.